src/mokka/src/main/java/pl/hycom/mokka/emulator/mock/MockConfigurationImportExportManager.java

```diff
@@ -91,17 +91,16 @@ public void loadMocks(InputStream stream) {
                     mc.setId(null);
                     mc.setDescription("[FILE UPLOAD] " + mc.getDescription());
                     mc.getConfigurationContent().setId(null);
-                    mockConfigurationManager.saveOrUpdateMockConfiguration(mc);
-
+                    mockConfigurationManager.createMockConfiguration(mc);
                 } else if (databaseMocks.size() == 1) {
                     mc.setId(databaseMocks.get(0).getId());
-                    mockConfigurationManager.saveOrUpdateMockConfiguration(mc);
+                    mockConfigurationManager.updateMockConfiguration(mc);
 
                 } else {
                     mc.setId(null);
                     mc.setEnabled(false);
                     mc.setDescription("[FILE UPLOAD] " + mc.getDescription());
-                    mockConfigurationManager.saveOrUpdateMockConfiguration(mc);
+                    mockConfigurationManager.updateMockConfiguration(mc);
```

Comment:
update where mock.id == null?

Should not we use `createMockConfiguration` instead?